### DISCUSSION TITLE
1022 Spam comments on your channel

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#619] [General]
I scanned the last 200 videos on your channel and found 1022 spam comments, you don't use the bot on your own channel? Lol

**Comment by KendallDoesCoding** [comment]
Log file of spam comments on @ThioJoe's channel:
[Spam_Log_2022-02-09_15-23-39.txt](https://github.com/ThioJoe/YT-Spammer-Purge/files/8033778/Spam_Log_2022-02-09_15-23-39.txt)

**Comment by TechStudent10** [comment]
I think would fit into "Show & Tell"
